src/vivarium_gates_nutrition_optimization_child/constants/results.py

```diff
@@ -51,10 +49,10 @@
 SEVERE_PEM_TRANSITION_COUNT_COLUMN_TEMPLATE = (
     "MEASURE_{SEVERE_PEM_TRANSITION}_event_count_AGE_GROUP_{AGE_GROUP}_SEX_{SEX}"
 )
-STUNTING_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_stunting_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_SEX_{SEX}"
-WASTING_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_wasting_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_BIRTH_WEIGHT_STATUS_{BIRTH_WEIGHT_STATUS}_SEX_{SEX}"
-WASTING_TRANSITION_COUNT_COLUMN_TEMPLATE = "MEASURE_{WASTING_TRANSITION}_event_count_AGE_GROUP_{AGE_GROUP}_BIRTH_WEIGHT_STATUS_{BIRTH_WEIGHT_STATUS}_SEX_{SEX}"
-UNDERWEIGHT_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_underweight_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_SEX_{SEX}"
+STUNTING_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_stunting_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_SEX_{SEX}_UNDERWEIGHT_STATE_{CGF_RISK_STATE}"
+WASTING_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_wasting_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_MAM_TREATMENT_{MAM_TREATMENT}_SAM_TREATMENT_{SAM_TREATMENT}_SEX_{SEX}_UNDERWEIGHT_STATE_{CGF_RISK_STATE}"
+WASTING_TRANSITION_COUNT_COLUMN_TEMPLATE = "MEASURE_{WASTING_TRANSITION}_event_count_AGE_GROUP_{AGE_GROUP}_MAM_TREATMENT_{MAM_TREATMENT}_SAM_TREATMENT_{SAM_TREATMENT}_SEX_{SEX}_UNDERWEIGHT_STATE_{CGF_RISK_STATE}"
+UNDERWEIGHT_STATE_PERSON_TIME_COLUMN_TEMPLATE = "MEASURE_child_underweight_{CGF_RISK_STATE_NUMERIC}_person_time_AGE_GROUP_{AGE_GROUP}_MAM_TREATMENT_{MAM_TREATMENT}_SAM_TREATMENT_{SAM_TREATMENT}_SEX_{SEX}_UNDERWEIGHT_STATE_{CGF_RISK_STATE}"
```

Comment:
Remind me, are CGF_RISK_STATE and CGF_RISK_STATE_NUMERIC not 1:1?

Reply:
Yup, they are 1:1!